frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/BaseResetBranchToRemoteAction.java

```diff
@@ -157,11 +178,9 @@ public void run(ProgressIndicator indicator) {
           // If `changes` is null the whole root will be refreshed
           GitUtil.refreshVfs(gitRepository.getRoot(), /* changes */ null);
         }
-      }
 
-      @Override
-      public void onSuccess() {
```

Comment:
No need to override `onSuccess` anymore?

Reply:
No, it's no needed coz it's run after each `run` method invocation regardless of checkout and reset operations success, so it's not what we want.